frontend/src/scenes/web-analytics/WebAnalyticsRecordings.tsx

```diff
@@ -0,0 +1,56 @@
+import { useValues } from 'kea'
+import { CompactList } from 'lib/components/CompactList/CompactList'
+import { RecordingRow } from 'scenes/project-homepage/RecentRecordings'
+import { SessionPlayerModal } from 'scenes/session-recordings/player/modal/SessionPlayerModal'
+import { sessionRecordingsPlaylistLogic } from 'scenes/session-recordings/playlist/sessionRecordingsPlaylistLogic'
+import { teamLogic } from 'scenes/teamLogic'
+import { urls } from 'scenes/urls'
+import { webAnalyticsLogic } from 'scenes/web-analytics/webAnalyticsLogic'
+
+import { SessionRecordingType } from '~/types'
+
+export function WebAnalyticsRecordingsTile(): JSX.Element {
+    const { replayFilters, webAnalyticsFilters } = useValues(webAnalyticsLogic)
+    const { currentTeam } = useValues(teamLogic)
+    const sessionRecordingsListLogicInstance = sessionRecordingsPlaylistLogic({
+        logicKey: 'webAnalytics',
+        filters: replayFilters,
+    })
+    const { sessionRecordings, sessionRecordingsResponseLoading } = useValues(sessionRecordingsListLogicInstance)
+    return (
+        <>
+            <SessionPlayerModal />
+            <CompactList
+                title="Recent recordings"
+                viewAllURL={urls.replay()}
```

Comment:
Might be nice to pass the filters through
```suggestion
                viewAllURL={urls.replay(ReplayTabs.Recent, replayFilters)}
```